I.pass-tests/10-paint-ford/paint-cars.js

```diff
@@ -0,0 +1,7 @@
+function paintCars(array, color) {
+    // I didn't understand what is required to be done so I hardcoded the index
```

Comment:
this is a tricky one 
they expect you to change the color of the Ford model (or first car) to the color they choose

Reply:
remember it should not change the original object colour
if you stuck let me know for more hints